Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelSaveTests.swift

```diff
@@ -18,16 +19,24 @@ final class SitterProfileViewModelSaveTests: XCTestCase {
         viewModel.surname = newSurname
         viewModel.phone = newPhone
         viewModel.bio = newBio
-        viewModel.pricePerHour = newPricePerHour
+        viewModel.pricePerHour = String(newPricePerHour)
 
+        // When
         viewModel.save()
 
-        let loadedSitter = viewModel.getCurrentSitter()
+        // Then
+        guard let data = KeyValueStorage(KeyValueStorage.Name.currentSitter)
+            .loadData(for: KeyValueStorage.Key.currentSitter) else {
+            return
+        }
+        guard let currentSitter = try? JSONDecoder().decode(Sitter.self, from: data) else {
+            return
+        }
```

Comment:
I think we should test it like a black box, not diving under the hood, because if decide to remove or replace `KeyValueStorage` we have to rewrite test as well.
Can we just create a new instance of ViewModel and verify that the data will be as expected?

You do it in the `testSitterProfileViewModelInitializedWithLastSavedData`

Reply:
@NadzeyaShpakouskaya 
But what do we expect from view model? Whether our changes will save and available after we will go to other screens and close the app, or our changes will be available on the `SitterProfileView`?

Reply:
In my opinion, the current view model shouldn't care about how and where the other views get the info. 
When we create a new instance of viewModel, we read the data from storage and that is our case. We want to check that data was updated.

Reply:
Fixed it, according suggestion of @NadzeyaShpakouskaya.
Please, check.